Automattic/wp-calypso
# Add "import" flow in the new site button

#### Proposed Changes

* We are adding a dropdown menu link to the new site button on the Sites Management Page. The link points to the new site import flow.

![Screen Shot 2022-09-15 at 10 22 43](https://user-images.githubusercontent.com/727413/190353744-63469da9-be14-4e61-b04a-0208f04033a5.png)


#### Testing Instructions



1. Open the `/sites` page
2. Click the arrow in the "Add new site" button
3. Click "Import an existing site"
4. Choose a new free domain
5. On the "Choose a plan" page, select a free plan

Expected result:
* the "What are your goals?" steps is displayed
* "Import my existing website content" is checked

#### Pre-merge Checklist






Related to https://github.com/Automattic/wp-calypso/issues/67534
